dev/addons/web-wallet.tf

```diff
@@ -19,14 +19,6 @@ resource "helm_release" "web_wallet" {
   namespace = kubernetes_namespace.addons.metadata[0].name
 
   depends_on = [kubernetes_secret.web_wallet_secret]
-}
```

Comment:
I think we want to keep both the helm releases. One is for the web wallet desktop layout, other one for the web wallet mobile layout. I think if we rename the web wallet mobile layout helm release, it should prevent the "already exists" error that you referenced [here](https://github.com/GaloyMoney/charts/issues/1078#issuecomment-1155065631).

Reply:
I have started to experiment with renaming, but then needed a new, matching named secret as well. Not sure if all be working well with 2 web-wallet servers connecetd to the same backend. Testing it is out of the scope of this PR I think. 

The `mobileLayout` is disabled in the values as well:
https://github.com/GaloyMoney/charts/blob/3f09cc0911720a0af01d48d28c95b534e1687222/dev/addons/web-wallet-mobile-values.yml#L2

i think this can be changed and applied quickly for testing.

Reply:
Running another build just in case it works with a simple change.